data/tmpl/values/content1.html

```diff
@@ -59,6 +61,7 @@
         <h2 class="token-awarded hidden" aria-hidden="true">Here’s your badge!</h2>
       </footer>
     </section>
+    <p class="step hidden">1</p>
```

Comment:
@schalkneethling just to grab the step for the sidebar. On the values side we only have 1 js file for every sidebar step so this was to compensate for that. I wasn't sure what the best means was for doing that. Forgot about the data tag :). I'll try that out.